Scroller/EditableWrapper.js

```diff
@@ -351,6 +406,13 @@ const EditableWrapper = (props) => {
 					const orders = finalizeOrders();
 					forwardCustom('onComplete', () => ({orders}))({}, editable);
 					reset();
+
+					setTimeout(() => {
+						announceRef.current.announce(
+							selectedItemLabel + $L('move complete'),
+							true
+						);
+					}, 300); // An arbitrary delay at a level that is not ignored by the new focus element
```

Comment:
Let's make a constant for this delay since it could be updated later if requested.